home/import_whatsapp_templates.py

```diff
@@ -119,6 +106,40 @@ def create_whatsapp_template_from_row(self, row: "ContentRow") -> None:
         template.save()
         return
 
+    def _get_or_init_whatsapp_template(
```

Comment:
I think this is more `_update_or_create`, since it either fetches the existing one and updates it, or creates a new one and returns it.